Resources/Public/JavaScript/Plugins/typo3image.js

```diff
@@ -286,13 +286,14 @@
                 + 'contentsLanguage=' + editor.config.contentsLanguage
                 + '&editorId=' + editor.id
                 + '&bparams=' + bparams.join('|'),
-            deferred = $.Deferred();
-
-        require(['TYPO3/CMS/Backend/Modal'], function (Modal) {
-            var $modal = Modal.advanced({
-                type: Modal.types.iframe,
-                title: editor.lang.common.image,
-                content: url,
+            deferred = $.Deferred(),
+            $modal;
+
+    require(['TYPO3/CMS/Backend/Modal'], function (Modal) {
```

Comment:
it seems there are unrelated white space changes - beside this, looks fine for me.

Reply:
Right, removed.